table/src/main/java/org/enso/table/serialization/Parser.java

```diff
@@ -0,0 +1,90 @@
+package org.enso.table.serialization;
```

Comment:
```suggestion
package org.enso.table.deserialization.csv;
```
There'll be other formats in future, and surely this is a deserialization operation?

Reply:
Having `.serialization.csv.Parser` and `.deserialization.csv.Printer` is a really clumsy package structure. Can change `serialization` to `format` if that suits you better.

Reply:
`format` is better, yes, but it has the wrong connotations for me. What about something like `exchange`, as it's really about exchanging data with other systems.